engine/src/main/scala/cromwell/webservice/routes/wes/WesRunRoutes.scala

```diff
@@ -0,0 +1,79 @@
+package cromwell.webservice.routes.wes
+
+import akka.actor.ActorRef
+import akka.http.scaladsl.model.{HttpHeader, StatusCodes}
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import akka.http.scaladsl.server.directives.RouteDirectives.complete
+import akka.util.Timeout
+import WesRunRoutes._
+import akka.stream.ActorMaterializer
+import com.typesafe.config.ConfigFactory
+import cromwell.webservice.routes.MetadataRouteSupport.metadataQueryRequest
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+
+trait WesRunRoutes {
+  implicit def materializer: ActorMaterializer
+
+  implicit val timeout: Timeout
+
+  val serviceRegistryActor: ActorRef
+
+  lazy val runRoutes: Route =
+      pathPrefix("ga4gh" / "wes" / "v1") {
+        concat(
+          pathPrefix("runs") {
+            concat(
+              pathEnd {
+                concat(
+                  get {
+                    parameters(("page_size".as[Int].?, "page_token".?)) { (pageSize, pageToken) =>
+                      completeCromwellResponse(listRuns(pageSize, pageToken, serviceRegistryActor))
+                    }
+                  }
+
+                )
+              }
+            )
+          }
+        )
+      }
+}
+
+object WesRunRoutes {
+
+  import akka.util.Timeout
+  import scala.concurrent.duration.FiniteDuration
+  import net.ceedubs.ficus.Ficus._
+
+  implicit lazy val duration: FiniteDuration = ConfigFactory.load().as[FiniteDuration]("akka.http.server.request-timeout")
+  implicit lazy val timeout: Timeout = duration
+
+  def extractAuthorizationHeader: HttpHeader => Option[HttpHeader] = {
+    case h: HttpHeader if h.name() == "Authorization" => Option(h)
+    case _ => None
+  }
```

Comment:
This function is unused. It is a bit alarming that we are skipping authorization? Or is this a copy-paste from the old `wes2cromwell` shim that is obsolete because the WES routes are within `/api/` and rely on proxy/CromIAM for authorization?

My high-level question is, with a CromIAM + Cromwell configuration, how are we assuring that user A cannot list runs belong to user B. If the answer is, "CromIAM does not have those routes so they can never be called," that is perfectly acceptable.